### DISCUSSION TITLE
buy claimblocks

### DISCUSSION THREAD

**Discussion by pedro751** [GriefPrevention/GriefPrevention#2107] [Ideas]
i would like to add the ability to buy claimblocks with a gui shop with the economy system

**Comment by Evidentsinger14** [comment]
Then configure a shop gui plugin to do that. Use the /acb command in your menu for it.

**Comment by pedro751** [comment]
but how cani make a shopgui that executes commands, i actually use
EconomyShopGui plugin, any way to do that on this plugin?

Charles DeLancey ***@***.***> escreveu em qui., 13/07/2023 às
13:50 :
